### ISSUE TITLE
Shield can't be used with keyboard

### ISSUE THREAD

**Issue by igorko** [flareteam/flare-game#318]
Using shield with shortcut or with keyboard navigation system (clicking twice "B" on ActionBar icon) doesn't work with Shield power because you should hold button to use shield. In the same time It works ok with mouse MAIN2: you hold shield when button is pressed and put down shield when button is released..
